### PR TITLE
Add Stock Component to Storbook

### PR DESCRIPTION
**Pull request by colefriedlaender**
Stock Component

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
LGTM 💯

File: components/Stock/Stock.module.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.stockName {
+  align-self: center;
+}
+.stockRate {
+  align-self: center;
+  justify-self: end;
+  color: var(--positive-color);
+}
+.amount {
+  font-size: 15px;
```
Comment (phlgr) [inline components/Stock/Stock.module.css]: These should be responsive

File: components/Stock/Stock.module.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.container {
+  background-color: var(--text-button-color);
+  height: 70px;
```
Comment (phlgr) [inline components/Stock/Stock.module.css]: Why does this need a fixed height?

File: components/Stock/Stock.module.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  align-self: center;
+  justify-self: end;
+  color: var(--positive-color);
+}
+.amount {
+  font-size: 15px;
+  color: var(--secundary-color);
+}
+.total {
+  font-size: 15px;
```
Comment (phlgr) [inline components/Stock/Stock.module.css]: Same as the above

**Review by phlgr — CHANGES_REQUESTED** [review]

**Pull request merged**